nixos/modules/system/activation/activation-script.nix

```diff
@@ -154,9 +154,15 @@ in
 
     system.activationScripts.tmpfs =
       ''
-        ${pkgs.utillinux}/bin/mount -o "remount,size=${config.boot.devSize}" none /dev
```

Comment:
Hm, would we actually need to remount here ourselves if we just added the special filesystems to the normal `/etc/fstab` as well, and let `nixos-rebuild` do the remounting if needed?